.github/workflows/build_and_test.yml

```diff
@@ -99,6 +101,12 @@ jobs:
           echo '::set-output name=type::scheduled'
           echo '::set-output name=envs::{"SKIP_MIMA": "true", "SKIP_UNIDOC": "true"}'
           echo '::set-output name=hadoop::hadoop3'
+        elif [ "${{ github.event.schedule }}" = "0 19 * * *" ]; then
+          echo '::set-output name=java::8'
+          echo '::set-output name=branch::branch-3.3'
+          echo '::set-output name=type::scheduled'
+          echo '::set-output name=envs::{"SCALA_PROFILE": "scala2.13"}'
+          echo '::set-output name=hadoop::hadoop3.3'
```

Comment:
According to `run-tests.py`, `hadoop3` instead of `hadoop3.3`?

https://github.com/apache/spark/blob/77339dc6a49d1d9d2a7a3aae966610acbe1a5d6e/dev/run-tests.py#L219-L222

Reply:
Oh, yes..